package.json

```diff
@@ -121,6 +147,19 @@
               "path": "walkthrough/testing.png",
               "altText": "Testing"
             }
+          },
```

Comment:
@digitarald  java getting started page will be opened on the top of welcome page? I think better to be opened on the right hand side. also, a good demo for side-by-side view.

Reply:
Correct, this is on the list for this release to allow. In my testing, the current Java Tools welcome page does not trigger any side by side views, so I would not block on this.